### ISSUE TITLE
Make Signup Page

### ISSUE THREAD

**Issue by batflarrow** [iiitv/CTF-frontend#5]

**Comment by orama254** [comment]
Greetings,
For the signup page is it ok to change the folder structure?

**Comment by Bianka-ui** [comment]
Hi! Can I work on this? @batflarrow

**Comment by batflarrow** [comment]
Yeah sure you can create folders of components and containers and work accordingly.

**Comment by batflarrow** [comment]
> Greetings @batflarrow , For the signup page is it ok to change the folder structure?

Yeah sure you can create folders of components and containers and work accordingly.

**Comment by orama254** [comment]
> > Greetings @batflarrow , For the signup page is it ok to change the folder structure?
> 
> Yeah sure you can create folders of components and containers and work accordingly.

Awesome, can I be assigned this and work on it?

**Comment by batflarrow** [comment]
> > > Greetings @batflarrow , For the signup page is it ok to change the folder structure?
> > 
> > 
> > Yeah sure you can create folders of components and containers and work accordingly.
> 
> Awesome, can I be assigned this and work on it?

Yeah sure you have now been assigned.

**Comment by batflarrow** [comment]
Any updates @orama254

**Comment by orama254** [comment]
> Any updates @orama254 ...?

Was thinking about starting with the folder structure first before creating the Signup Page @batflarrow

**Comment by batflarrow** [comment]
> > Any updates @orama254 ...?
> 
> Was thinking about starting with the folder structure first before creating the Signup Page @batflarrow

As I suggested there will be two folders Components and Containers Entire SIgnup page will be placed in the Containers folder under Containers folder and  the various components for the page will be placed in the components folder under the Signup folder.

**Comment by orama254** [comment]
> > > Any updates @orama254 ...?
> > 
> > 
> > Was thinking about starting with the folder structure first before creating the Signup Page @batflarrow
> 
> As I suggested there will be two folders Components and Containers Entire SIgnup page will be placed in the Containers folder under Containers folder and the various components for the page will be placed in the components folder under the Signup folder.

Okay, maybe I can create a PR with the suggested folder structure first then you can review it before creating the signup page. Would that be ok?

**Comment by batflarrow** [comment]
> > > > Any updates @orama254 ...?
> > > 
> > > 
> > > Was thinking about starting with the folder structure first before creating the Signup Page @batflarrow
> > 
> > 
> > As I suggested there will be two folders Components and Containers Entire SIgnup page will be placed in the Containers folder under Containers folder and the various components for the page will be placed in the components folder under the Signup folder.
> 
> Okay, maybe I can create a PR with the suggested folder structure first then you can review it before creating the signup page. Would that be ok?

Yeah ok I won't merge it but if you want you can make a pr and then update the signup page in that branch itself.

**Comment by orama254** [comment]
> > > > > Any updates @orama254 ...?
> > > > 
> > > > 
> > > > Was thinking about starting with the folder structure first before creating the Signup Page @batflarrow
> > > 
> > > 
> > > As I suggested there will be two folders Components and Containers Entire SIgnup page will be placed in the Containers folder under Containers folder and the various components for the page will be placed in the components folder under the Signup folder.
> > 
> > 
> > Okay, maybe I can create a PR with the suggested folder structure first then you can review it before creating the signup page. Would that be ok?
> 
> Yeah ok I won't merge it but if you want you can make a pr and then update the signup page in that branch itself.

you can look at the suggestion on PR #8 to see if it comes close to the suggestion

**Comment by batflarrow** [comment]
> > > > > > Any updates @orama254 ...?
> > > > > 
> > > > > 
> > > > > Was thinking about starting with the folder structure first before creating the Signup Page @batflarrow
> > > > 
> > > > 
> > > > As I suggested there will be two folders Components and Containers Entire SIgnup page will be placed in the Containers folder under Containers folder and the various components for the page will be placed in the components folder under the Signup folder.
> > > 
> > > 
> > > Okay, maybe I can create a PR with the suggested folder structure first then you can review it before creating the signup page. Would that be ok?
> > 
> > 
> > Yeah ok I won't merge it but if you want you can make a pr and then update the signup page in that branch itself.
> 
> you can look at the suggestion on PR #8 to see if it comes close to the suggestion

Yeah sure the folder structure is fine you may move ahead with the task.

**Comment by orama254** [comment]
hey @batflarrow 

wanted to know which font was used on the design

##### screenshot:
![ctf font](https://user-images.githubusercontent.com/50571688/136164349-32f3ba0a-874e-42ef-b90f-0388a1e9df9b.jpg)

**Comment by poojavirgo** [comment]
@batflarrow  can you please assign this issue to me?

**Comment by batflarrow** [comment]
> > hey @batflarrow
> > wanted to know which font was used on the design
> > ##### screenshot:
> > ![ctf font](https://user-images.githubusercontent.com/50571688/136164349-32f3ba0a-874e-42ef-b90f-0388a1e9df9b.jpg)
> 
> It is present in the figma design :)

Also @orama254 You have until the end of today to complete this issue or else you will be unassigned.

**Comment by poojavirgo** [comment]
Thanks

**Comment by orama254** [comment]
> > > hey @batflarrow
> > > wanted to know which font was used on the design
> > > ##### screenshot:
> > > ![ctf font](https://user-images.githubusercontent.com/50571688/136164349-32f3ba0a-874e-42ef-b90f-0388a1e9df9b.jpg)
> > 
> > 
> > It is present in the figma design :)
> 
> Also @orama254 You have until the end of today to complete this issue or else you will be unassigned.

hey @batflarrow unfortunately I wont be able to accomplish this by the end of the day. You can assign @poojavirgo the task

I'll try and contribute later on other issues/features of the project
